doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,31 @@
+= Transforming coordinate systems
+
+Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+
+//TODO: Should we add one or more sentences about the mathematical operations involved?
+
+**Corresponding messages**
+
+``GroundTruth::moving_object::base::position``::
+Defines the position of the vehicle’s reference point in world coordinates.
+In Open Simulation Interface, the position of an object is defined by the coordinates of the center of the object's bounding box.
+
+``GroundTruth::moving_object::base::orientation``::
+Defines the orientation of the vehicle’s reference point in world coordinates.
+
+``GroundTruth::moving_object::vehicle_attributes::bbcenter_to_rear``::
+The vector pointing from the vehicle’s reference point to the middle of the rear axle under neutral load conditions in vehicle coordinates.
+
```

Comment:
@ThomasNaderBMW, @pmai, @clemenshabedank 
What about other objects, like stationary objects, traffic signs and lights?
To be discussed in the next CCB meeting.

Reply:
@pasched to be done. To be discussed in CCB today.

Reply:
OSI CCB:
- Review in OSI CCB: Should be fine like this.